src/popup/modules/QrLib/QrError.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Errors in QR code generation
+ *
+ * @module QrLib/QrErrors
+ */
+
+export class DataOverflowError extends Error {
+    constructor(...params) {
+        super(...params);
+        this.message = "data overflow error";
```

Comment:
Also the message may be more human-readable:
```suggestion
        this.message = "The QR code was given too much data.";
```